recipes/binutils/all/test_package/conanfile.py

```diff
@@ -1,22 +1,44 @@
-from conans import ConanFile, tools
+from conan import ConanFile, tools
 from io import StringIO
 import os
+import json
+import sys
 
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
+    test_type = "explicit"
```

Comment:
```suggestion
    generators = "VirtualBuildEnv"
    test_type = "explicit"
```
Need to generate scripts with environment variables.